File: Coding/03_Beginner/Full_search/JavaScript/5.js
const fs = require("fs");
const input = fs.readFileSync(0, "utf8").trim().split(/\s+/);

const N = Number(input[0]);
const A = input.slice(1).map(Number);

let ans = -1;
for (let i = 0; i < N; i++){
    if (A[i] > ans){
        ans = A[i];
    }
}

console.log(ans);

File: Coding/03_Beginner/set_dict/Python/5.py
N = int(input())
A = [0] + list(map(int,input().split()))

cnt = {}
for i in range(N+1):
    for j in range(N+1):
        for k in range(N+1):
            score = A[i]+A[j]+A[k]
            cnt[score] = 1

print(len(cnt))